ansible/roles/docker/tasks/main.yml

```diff
@@ -69,8 +69,8 @@
   tags: configure
 
 - name: Add any insecure registrys to docker config
-  lineinfile: dest={{ docker_config_dir }}/docker regexp=^INSECURE_REGISTRY= line=INSECURE_REGISTRY="{% for reg in insecure_registrys %}--insecure-registry={{ reg }} {% endfor %}"
-  when: insecure_registrys is defined and insecure_registrys > 0
+  lineinfile: dest={{ docker_config_dir }}/docker regexp="^INSECURE_REGISTRY=" line="INSECURE_REGISTRY={% for reg in insecure_registrys %}--insecure-registry={{ reg }} {% endfor %}"
```

Comment:
I prefer transparency over file scoped coding style. If you don't mind, can you alter all other lineinfile occurrences and squash the commits? I will merge then. Thanks.